src/piprules/bazel.py

```diff
@@ -69,12 +78,63 @@ def _create_base_package_build_file(self):
             )
         """).lstrip().format(
             name=self.library_name,
-            deps=_create_string_list(dep.label for dep in self.library_dependencies),
+            deps=_create_string_list(
+                dep.label for dep in self.library_dependencies),
         )
 
         with open(self.base_package_build_file_path, mode="w") as build_file:
             build_file.write(contents)
 
+    def _generate_bin_build_file(self):
+        contents = "".join([
+            textwrap.dedent("""
+            py_binary(
+                name = "{rule}",
+                srcs = ["bin/{entry_point}.py"],
+                deps = [":{library_name}"],
```

Comment:
we need to provide `main = "bin/{entry_point}.py"` here, because in the case when `{rule}` has `bin-` prepended it won't match automatically.

Reply:
Added. Thank for catching that.